README.md

```diff
@@ -71,28 +69,27 @@ Example usage:
     Run `git push heroku master` to create a new release using these buildpacks.
 
     $ cat Procfile
-    web:    bin/start-pgbouncer-stunnel bundle exec unicorn -p $PORT -c ./config/unicorn.rb -E $RACK_ENV
+    web:    bin/start-pgbouncer bundle exec unicorn -p $PORT -c ./config/unicorn.rb -E $RACK_ENV
```

Comment:
i'm concerned about us just renaming that command; i would be happier if we kept two scripts:
* bin/start-pgbouncer
as well as
* bin/start-pgbouncer-stunnel that does exactly what bin/start-pgbouncer does (if there is a way of making it not too intrusive, add a log?)

We could go ask the languages team if there is a way to warn customers who use it that we are renaming the command? (we can definitely look at what buildpacks folks are using; I however don't know if the Procfile counts as metadata that we can look at, or customer code that we cannot look at)
I am concerned folks are going to have the buildpack configured to pull direct from master of this repo and we are going to break their applications when we push this change.

Reply:
thank you for the explanation! if it's backwards-compatible then this PR is fine by me :) 👍

Reply:
Our prod just went down after upgrading to Cedar-18 with this error:
```
Using bin/start-pgbouncer-stunnel, which is a deprecated init script. Use bin/start-pgbouncer instead.
...
vendor/pgbouncer/bin/pgbouncer: error while loading shared libraries: libevent-2.0.so.5: cannot open shared object file: No such file or directory
```

That was pretty sad :-/